charms/worker/k8s/src/token_distributor.py

```diff
@@ -174,19 +147,38 @@ def create(self, name: str, token_type: ClusterTokenType) -> SecretStr:
             username=f"system:cos:{name}", groups=["system:cos"]
         )
 
-    def revoke(self, name: str, ignore_errors: bool):
+    def revoke(self, _name: str, secret: Optional[ops.Secret], ignore_errors: bool):
```

Comment:
The `_name` argument in the signature seems unused.  If we are not using it, let's remove it to clean up the method signature.

Reply:
Gotcha, makes sense. Thanks!